GW2EIEvtcParser/EncounterLogic/Strikes/EndOfDragons/HarvestTemple.cs

```diff
@@ -637,6 +637,21 @@ internal override void ComputeNPCCombatReplayActors(NPC target, ParsedEvtcLog lo
                         }
                     }
                     //
+                    EffectGUIDEvent orbExploded = log.CombatData.GetEffectGUIDEvent(EffectGUIDs.HarvestTempleOrbExplosion);
+                    if (orbExploded != null)
+                    {
+                        IReadOnlyList<EffectEvent> orbEffects = log.CombatData.GetEffectEventsByEffectID(orbExploded.ContentID);
+                        knownEffectsIDs.Add(orbExploded.ContentID);
+                        foreach (EffectEvent orbEffect in orbEffects)
+                        {
+                            int duration = 3000;
```

Comment:
Are the duration and radius values accurate? Or is it simply an indication?

Reply:
Is this something you wish to do in this PR or at a later date? I don't think it's worth blocking the PR for 100 units on something that is this big.